src/antlr/Parser.g

```diff
@@ -1535,10 +1528,11 @@ listUsersStatement returns [ListRolesStatement stmt]
     ;
 
 /**
- * CREATE ROLE [IF NOT EXISTS] <rolename> [ [WITH] option [ [AND] option ]* ]
+ * CREATE [GENERATED] ROLE [IF NOT EXISTS] <rolename> [ [WITH] option [ [AND] option ]* ]
```

Comment:
Does it make sense for `[GENERATED]` and `[IF NOT EXISTS]` to coexist?

Reply:
I added throwing `SyntaxException` if that occurs.